src/components/CardForm/CardForm.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  CardNumberInput,
+  CardExpirationDateInput,
+  CardOwnerNameInput,
+  CardSecurity,
+  CardPasswordInput,
+} from 'components/CardFormInput';
+import { VirtualKeyBoard } from 'components/common/VirtualKeyBoard';
+import type { CardFormProps } from './CardForm.types';
+import { getCardCompnayColor } from 'utils/Card';
+import { isValidCardNumber, isValidExpirationDate } from 'utils/InputValidation';
+
+const CardForm = ({
+  cardUI,
+  refs,
+  onCardNumberInput,
+  onExpireMonthInput,
+  onExpireYearInput,
+  onOwnerNameInput,
+}: CardFormProps) => {
+  const cardColor = getCardCompnayColor(cardUI.company);
+  return (
+    <div>
+      <CardNumberInput
+        onChange={onCardNumberInput}
+        isValid={cardUI.cardNumbers.length === 0 ? true : isValidCardNumber(cardUI.cardNumbers)}
+        fontColor={cardColor}
+        refs={refs}
+      />
+      <CardExpirationDateInput
+        onChangeMonth={onExpireMonthInput}
+        onChangeYear={onExpireYearInput}
+        fontColor={cardColor}
+        refs={refs}
+        isValidMonth={cardUI.expireDateMonth.length === 0 ? true : isValidExpirationDate(cardUI.expireDateMonth)}
+        isValidYear={cardUI.expireDateYear.length === 0 ? true : isValidExpirationDate(cardUI.expireDateYear)}
```

Comment:
```suggestion
        isValidMonth={isValidExpirationDate(cardUI.expireDateMonth)}
        isValidYear={isValidExpirationDate(cardUI.expireDateYear)}
```

너무 사용하기 어려운 로직으로 만들어놨네요.
이런 유효성 검사를 하신다면 모조리 위임하시는게 맞지 않을까요?